js/simpletextbox/Index.svelte

```diff
@@ -0,0 +1,129 @@
+<svelte:options accessors={true} />
+
+<script lang="ts">
+	import type { Gradio } from "@gradio/utils";
+	import { BlockTitle } from "@gradio/atoms";
+	import { Block } from "@gradio/atoms";
+	import { StatusTracker } from "@gradio/statustracker";
+	import type { LoadingStatus } from "@gradio/statustracker";
+	import { tick } from "svelte";
+
+	export let gradio: Gradio<{
+		change: string;
+		submit: never;
+		input: never;
+	}>;
+	export let label = "Textbox";
+	export let elem_id = "";
+	export let elem_classes: string[] = [];
+	export let visible = true;
+	export let value = "";
+	export let placeholder = "";
+	export let show_label: boolean;
+	export let scale: number | null = null;
+	export let min_width: number | undefined = undefined;
+	export let loading_status: LoadingStatus | undefined = undefined;
+	export let value_is_output = false;
+	export let mode: "static" | "interactive";
+	export let rtl = false;
+
+
+	let el: HTMLTextAreaElement | HTMLInputElement;
+	const container = true;
+
+	function handle_change(): void {
+		gradio.dispatch("change", value);
+		if (!value_is_output) {
+			gradio.dispatch("input");
+		}
+	}
+
+	async function handle_keypress(e: KeyboardEvent): Promise<void> {
+		await tick();
+		if (e.key === "Enter") {
+			e.preventDefault();
+			gradio.dispatch("submit");
+		}
+	}
+
+	$: if (value === null) value = "";
+
+	$: value, handle_change();
```

Comment:
I always found this syntax a bit confusing. I don't know if there's a way to rewrite more clearly in svelte. Maybe its worth an explanatory comment.

Reply:
We want `handle_change` to run when the value changes so I guess we can do

```ts
$:   handle_change(value)
```

Do you have thoughts on this @pngwn ?

Reply:
Yep. That's a better way to do it if it makes sense for the function we are calling.

Reply:
Ok will push up. Verified it works!

Reply:
But if we make the change above, handle_change() doesn't utilize the value at all

Reply:
This change: 

<img width="952" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/ea0f2f85-5ddc-406a-9008-58744b5e0c9d">

Reply:
I think that's ok?

Reply:
It won't cause any issues but it doesn't seem like good practice to pass a value into the function that it won't use. I personally think adding a comment for clarity to the original syntax, but its okay either way.

Reply:
let's use a comment!

Reply:
It would also be a type or linting error, so comment works well.